helm_deploy/hmpps-template-typescript/values.yaml

```diff
@@ -1,5 +1,6 @@
 generic-service:
   nameOverride: hmpps-template-typescript
+  serviceAccountName: "UNASSIGNED" # the IRSA service account name for accessing AWS resources
```

Comment:
is this going to prevent the pod from starting in development when this is merged?

Reply:
I'm not 100% sure if it will fail to deploy, but it's a good point and I've commented it out for now.

If there's value in integrating with audit, it could be done, but it would mean providing a valid service account name which might be problematic when cloning the project if teams don't realise it needs to be updated.

It might be better if it's left as a comment with the "UNASSIGNED" value since I guess teams are looking for "UNASSIGNED" references to update already?